apps/web/app/(app)/environments/[environmentId]/settings/lookandfeel/components/EditPlacement.tsx

```diff
@@ -58,18 +60,21 @@ export function EditPlacement({ product }: EditPlacementProps) {
           {placements.map((placement) => (
             <div key={placement.value} className="flex items-center space-x-2 whitespace-nowrap">
               <RadioGroupItem id={placement.value} value={placement.value} disabled={placement.disabled} />
-              <Label
-                htmlFor={placement.value}
-                className={cn(placement.disabled ? "cursor-not-allowed text-slate-500" : "text-slate-900")}>
+              <Label htmlFor={placement.value} className="text-slate-900">
                 {placement.name}
               </Label>
             </div>
           ))}
         </RadioGroup>
-        <div className="relative ml-8 h-40 w-full rounded bg-slate-200">
+        <div
+          className={cn(
+            clickOutside === "disallow" ? "cursor-not-allowed" : "",
+            "relative ml-8 h-40 w-full rounded",
+            overlayStyle
```

Comment:
I would prefer if we include the `overlayStyle` inline here as it's not reused in different places and makes it more complicated to read/debug the code & styles since we need to look in to different places. But it's feedback, not a dealbreaker :-)